dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Skills.Memory.VectorDB.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
`<GenerateAssemblyInfo>false</GenerateAssemblyInfo>`
This is specified twice, but should not be specified at all if it's to be packaged as nuget. Why is this here?